app/components/channel_drawer/channels_list/channel_item.js

```diff
@@ -26,9 +26,9 @@ export default class ChannelItem extends PureComponent {
 
     onPress = () => {
         const {channel, onSelectChannel} = this.props;
-        setTimeout(() => {
+        requestAnimationFrame(() => {
```

Comment:
if we don't do this the affect of the item being pressed won't show up

Reply:
Doesn't need to be added in this PR, but we need to make use of this library for setTimeouts in components. 

https://www.npmjs.com/package/react-timeout

It will handle the this.mounted = true/false for us and stop us from setting state after a component is unmounted